### PR TITLE
Edit Static Pages and Add Contact

### PR DESCRIPTION
**Pull request by theduc151**
## Checklist
- [ ] This is bug/feature (delete incorrect one)
- [ ] Title including ticket number and title

## What this PR do?
Explain clearly and briefly what this PR does. Make a meaningful sentence instead of "this PR solve ticket #xxx"

## Context/Background
More information on the issue that you want to provide, related issues and PRs you have to write information to ask orally or in informal place too.

## Screenshots / Demo & Instruction for testing (if applicable)
image
image

## References
- Blog posts / web pages relevant to this issue (which you get your solution / idea from and it may help others to understand the issue quickly)
- Do we need update our knowledge base / project wiki?

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -12,4 +12,6 @@
  *
  *= require_tree .
  *= require_self
+ *= require font-awesome
  */
+
```
Comment (rikai-tra) [inline app/assets/stylesheets/application.css]: endline

File: .byebug_history [hunk 2]
```diff
@@ -0,0 +1 @@
+c
```
Comment (rikai-tra) [inline .byebug_history]: không cho file này lên

File: app/views/layouts/_account.slim [hunk 3]
```diff
@@ -0,0 +1,9 @@
+li.dropdown.navbar-right
+  a.dropdown-toggle.mr-5 data-toggle="dropdown" type="button"
+    | Account
+    b.caret
+  ul.dropdown-menu
+    li= link_to "Profile", user_path(current_user)
+    li= link_to "Settings", edit_user_path(current_user)
+    li= link_to "Help", help_path
+    li= link_to "Log out", logout_path, method: :delete
```
Comment (rikai-tra) [inline app/views/layouts/_account.slim]: endline

File: config/database.yml [hunk 4]
```diff
@@ -3,7 +3,7 @@ default: &default
   host: 127.0.0.1
   pool: 5
   username: postgres
-  password: password
+  password: root
```
Comment (rikai-hai) [inline config/database.yml:6]: đưa vào gitignore

Comment (theduc151) [reply config/database.yml:6]: image
đưa vào như vậy à ???

**Pull request merged**